src/plotman/manager.py

```diff
@@ -81,9 +95,9 @@ def maybe_start_new_plot(dir_cfg: plotman.configuration.Directories, sched_cfg:
     youngest_job_age = min(jobs, key=job.Job.get_time_wall).get_time_wall() if jobs else MAX_AGE
     global_stagger = int(sched_cfg.global_stagger_m * MIN)
     if (youngest_job_age < global_stagger):
-        wait_reason = 'stagger (%ds/%ds)' % (youngest_job_age, global_stagger)
+        wait_reason = 'global stagger (%ds/%ds)' % (youngest_job_age, global_stagger)
     elif len(jobs) >= sched_cfg.global_max_jobs:
-        wait_reason = 'max jobs (%d) - (%ds/%ds)' % (sched_cfg.global_max_jobs, youngest_job_age, global_stagger)
+        wait_reason = 'global max jobs reached (%d)' % (sched_cfg.global_max_jobs)
```

Comment:
The stagger seconds was added not too long ago.  I personally plan on going the other way and considering how to indicate the status of all restrictions at all times.  Was there a particular need to remove this?

```suggestion
        wait_reason = 'global max jobs reached (%d) - (%ds/%ds)' % (sched_cfg.global_max_jobs, youngest_job_age, global_stagger)
```

Side note, and no need to change it now, but this PR probably should have been at least four.  Offhand, the new per-tmp phase stagger, the addition of global, removal of the time-stagger indication here, and moving of the overrides to scheduling.  Just a thought for next time.  Small PRs are quicker to verify and merge (and undo if needed).